extras/speaker-recognition/src/simple_speaker_recognition/api/routers/identification.py

```
@@ -281,48 +283,123 @@ async def diarize_identify_match(
+
     The transcript_data should be a JSON string containing:
     {
         "words": [{"word": "hello", "start": 1.23, "end": 1.45}, ...],
         "text": "full transcript text"
     }
+
+    Maximum audio duration: 2 hours (7200 seconds)
+    Files longer than max_diarize_duration (default 60s) are automatically chunked
     """
-    log.info(f"Processing diarize-identify-match request: {file.filename}")
+    log.info(f"Processing diarize-identify-match request")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)